[FW][FIX] l10n_ar_account_withholding: withholding type requerido

fw-bot-adhoc opened the pull request:
Ticket: 81960
Este cambio va de la mano con el de este pr https://github.com/ingadhoc/account-payment/pull/547

Forward-Port-Of: ingadhoc/odoo-argentina#941
Forward-Port-Of: ingadhoc/odoo-argentina#934

pablohmontenegro committed:
[FIX] l10n_ar_withholding_ux: withholding type requerido

FW of change made in l10n_ar_account_withholding module
but now applied to a more proper module (account_withholding_automatic
was deprecated, and new l10n_ar_withholding_ux is the one used)

Ticket: 81960

Este cambio va de la mano con el de este pr https://github.com/ingadhoc/account-payment/pull/547

X-original-commit: 663826b7b0db5c6ddb03dd986e2001fb3e910d91

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/1032).

fw-bot-adhoc commented:
@pablohmontenegro @zaoral cherrypicking of pull request ingadhoc/odoo-argentina#934 failed.

stdout:
```
CONFLICT (modify/delete): l10n_ar_withholding_ux/views/account_tax_view.xml deleted in HEAD and modified in 49d07bf9 ([FIX] l10n_ar_withholding_ux: withholding type requerido). Version 49d07bf9 ([FIX] l10n_ar_withholding_ux: withholding type requerido) of l10n_ar_withholding_ux/views/account_tax_view.xml left in tree.
CONFLICT (modify/delete): l10n_ar_withholding_ux/models/account_tax.py deleted in HEAD and modified in 49d07bf9 ([FIX] l10n_ar_withholding_ux: withholding type requerido). Version 49d07bf9 ([FIX] l10n_ar_withholding_ux: withholding type requerido) of l10n_ar_withholding_ux/models/account_tax.py left in tree.
CONFLICT (modify/delete): l10n_ar_withholding_ux/__manifest__.py deleted in HEAD and modified in 49d07bf9 ([FIX] l10n_ar_withholding_ux: withholding type requerido). Version 49d07bf9 ([FIX] l10n_ar_withholding_ux: withholding type requerido) of l10n_ar_withholding_ux/__manifest__.py left in tree.

```

stderr:
```
14:55:14.771426 git.c:444               trace: built-in: git cherry-pick 49d07bf99864bb2989921cee07500b8d357c9c8c
error: could not apply 49d07bf9... [FIX] l10n_ar_withholding_ux: withholding type requerido
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
